test/system/src/test/java/io/pravega/test/system/AbstractReadWriteTest.java

```diff
@@ -92,8 +92,8 @@
         final CompletableFuture<Void> newWritersComplete = new CompletableFuture<>();
         final CompletableFuture<Void> readersComplete = new CompletableFuture<>();
         final List<CompletableFuture<Void>> txnStatusFutureList = synchronizedList(new ArrayList<>());
-        final ConcurrentSet<UUID> committingTxn = new ConcurrentSet<>();
-        final ConcurrentSet<UUID> abortedTxn = new ConcurrentSet<>();
+        final ConcurrentSkipListSet<UUID> committingTxn = new ConcurrentSkipListSet<>();
```

Comment:
As I understand the reason for this change is that io.netty.util.internal.ConcurrentSet is deprecated ?

Reply:
Yes, correct.